backend/src/main/java/wooteco/prolog/studylog/ui/RssFeedController.java

```diff
@@ -0,0 +1,23 @@
+package wooteco.prolog.studylog.ui;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.View;
+import wooteco.prolog.studylog.application.RssFeedService;
+
+@Controller
+@RequestMapping("/rss-feeds")
+public class RssFeedController {
+
+    private final RssFeedService rssFeedService;
+
+    public RssFeedController(RssFeedService rssFeedService) {
+        this.rssFeedService = rssFeedService;
+    }
+
+    @GetMapping
+    public View getRssFeed() {
+        return rssFeedService;
+    }
```

Comment:
Model을 이용하여 값을 넘겨줘도 좋을 것 같습니다!
```suggestion
    public RssFeedView getRssFeed(Model model) {
        List<RssFeedResponse> feeds = studylogService.readRssFeeds();
        model.addAttribute("feeds", feeds);
        return rssFeedView;
    }
```